### ISSUE TITLE
postgresql url with sslmode get error in 2.0.0-beta.5 verison

### ISSUE THREAD

**Issue by leohxj** [prisma/migrate#457]
## Bug description
I am updating from beta.2 to beta.5.
run `npx prisma2 migrate save --experimental` get below errors:

![image](https://user-images.githubusercontent.com/1506900/82790462-aba59b80-9e9e-11ea-95b7-673d250dff0c.png)

when I remove the `sslmode=prefer`, it works well.

## How to reproduce
I am use the postgresql url is ended with `sslmode=perfert`, the whole url is in the `.env` file:

```
# Executing Migration, Studio and seed locally
POSTGRESQL_URL=postgresql://prisma:topsecret@localhost:5432/ispacegen?schema=public&sslmode=prefer
```

## Expected behavior
can generate the new sql migration.

## Environment & setup

- OS: [e.g. Mac OS, Windows, Debian, CentOS, ...]
Mac OS
- Database: [PostgreSQL, MySQL, MariaDB or SQLite]
PostgreSQL
- Prisma version: [Run `prisma -v` to see your Prisma version]
2.0.0-beta.5
- Node.js version: [Run `node -v` to see your Node.js version]
v10.15.0

**Comment by pantharshit00** [comment]
Hi,

Can you please tell me the error report id? Or please provide enough details so that I can reproduce this issue, thanks!

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes](https://github.com/prisma/prisma/releases/tag/2.13.0)), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma](https://github.com/prisma/prisma/issues/new/choose).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue](https://github.com/prisma/prisma/issues/4531).
